File: Python/problem002.py
#By considering the terms in the Fibonacci sequence whose values do not exceed four million, 
# find the sum of the even-valued terms

n1 = n2 = 1
n3 = 0
sum = 0
while True:
    n3 = n1 + n2
    if n3 < 4*10**6:
        if n3 % 2 == 0:
            sum += n3
        n1 = n2
        n2 = n3
        n3 = 0
    else:
        break

print(sum)

File: C/problem002.c
/* #By considering the terms in the Fibonacci sequence whose values do not exceed four million, 
find the sum of the even-valued terms */

#include <stdio.h>
#include <math.h>

int main()
{   
    long long term1 = 1, term2 = 1,term3 = term1+term2;
    long long sum = 0;
    long long bound = pow(10, 6);
    while (term3 < 4*bound)
    {
        if (term3 %2 == 0)
            sum += term3;
        term1 = term2;
        term2 = term3;
        term3 = term1 + term2;
    }
    printf("%lld", sum);

    return 0;
}   